doc/source/manual/xtrabackup_sst.rst

```diff
@@ -239,30 +272,21 @@ example: ::
 
 .. option:: sst_special_dirs
 
-     :Values: 0, 1
-     :Default: 1
-
-This option was introduced in |Percona XtraDB Cluster| :rn:`5.6.15-25.2` to
-enable XtraBackup SST to support :variable:`innodb_data_home_dir` and
-:variable:`innodb_log_home_dir` variables in the configuration file. |Percona
-Xtrabackup| 2.1.6 or later is required for this option to work. **NOTE:** This
-variable has been removed and deprecated in |Percona XtraDB Cluster|
-:rn:`5.6.22-25.8`.
+     :Values: ``0``, ``1``
```

Comment:
@kennt-percona suggested this option is now removed. Why are we still carrying it here in doc ?